plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java

```diff
@@ -615,6 +616,7 @@ public void testCreateTableWithProperties()
                         "   d varchar(10)\n" +
                         ")\n" +
                         "WITH (\n" +
+                        "   bloomfilter = 'ROW',\n" +
```

Comment:
where does this come from?

Reply:
it's now being returned as a default table property (bloomfilter with all its values was already supported)